### PR TITLE
docs(models): add section on MongoDB Views

### LINKED ISSUE #5694: Is there a way we can handle views with mongoose !?
***feature***

**What is the current behavior?**
currently there is a plugin, but that only selects few fields, and doesn't work on multiple models

**What is the expected behavior?**
https://docs.mongodb.com/manual/reference/method/db.createView/#db.createView

using mongoDB 3.4

### PR DESCRIPTION
**Pull request by vkarpov15**
Fix #5694

**Summary**

Right now there isn't any info on the docs on how to use Mongoose with MongoDB views. Admittedly I haven't used views in production, so I can't say I'm an expert on the subject. But this new docs section at least demonstrates how to get started with creating a new View in Mongoose.

I also replaced h2 with h1 and h3 with h2 in the models docs, because it doesn't make much sense for the main title of the page to be an h2.

**Examples**

### CODE REVIEW

File: docs/models.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ reading documents from the underlying MongoDB database.
 * [Deleting](#deleting)
 * [Updating](#updating)
 * [Change Streams](#change-streams)
+* [Views](#views)
 
 <h3 id="compiling"><a href="#compiling">Compiling your first model</a></h3>
```
Comment (hasezoey) [inline docs/models.md:16]:
```suggestion
<h2 id="compiling"><a href="#compiling">Compiling your first model</a></h3>
```

i think this was missed when changing to h2

**Review by hasezoey — APPROVED** [review]
LGTM

File: docs/models.md (hunk 1 above)
Comment (AbdelrahmanHafez) [inline docs/models.md:16]:
```suggestion
<h2 id="compiling"><a href="#compiling">Compiling your first model</a></h2>
```

Comment (vkarpov15) [reply docs/models.md]: already fixed by @hasezoey . Thanks for catching this though.

Comment (hasezoey) [reply docs/models.md]: actually, i just noticed that in my suggestions i did not change the closing tag

**Pull request merged**